src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -1475,6 +1500,7 @@ define(function(require) {
 							data: self.portWizardNormalizePortRequestData(args.data.request)
 						},
 						success: function(portRequest) {
+							_.set(args, 'data.request.id', portRequest.id);
```

Comment:
I believe there is a store for this, so maybe you should save the whole payload:
```suggestion
							self.portWizardSet('portRequest', portRequest);
```

Reply:
Never mind, doesn't seem like the wizard's steps use the store.

Reply:
Yeah, it does not seem to use it.